dags/msg2teams.py

```diff
@@ -50,11 +53,11 @@ def main():
 
 # script execution entry point
 logging.basicConfig(
-    filename='msg2msteams.log',
-    filemode='a',
+#    filename='msg2msteams.log',
```

Comment:
it's better not to store comments with code inside git - in case old version is required, you could just view it in a history and restore